modules/core/src/v2/coins/eth.ts

```diff
@@ -224,11 +235,30 @@ interface FeeEstimate {
   feeEstimate: number;
 }
 
-interface TransactionPrebuild extends BaseTransactionPrebuild {
+export interface TransactionPrebuild extends BaseTransactionPrebuild {
   hopTransaction?: HopPrebuild;
   buildParams: {
     recipients: Recipient[];
   };
+  recipients?: TransactionRecipient[];
+  nextContractSequenceId?: string;
+  gasPrice?: number;
+  gasLimit?: number;
+  isBatch?: boolean;
+  coin?: string;
+  token?: string;
+  // eth2DepositInfo?: any;
+}
+
+interface EthTransactionParams extends TransactionParams {
```

Comment:
Also might be worth leaving a TODO for the london hard fork task that will likely need to update this

Reply:
Added TODO comment